Spec/RealtimeClientConnection.swift

```diff
@@ -3583,6 +3623,51 @@ class RealtimeClientConnection: QuickSpec {
                     }
                 }
             }
+
+            it("should abort reconnection with new token if the server has requested it to authorise and after it the connection has been closed") {
```

Comment:
None. That was something I notice on the test suite where the client gets connected when it's waiting for the `authorize` and the client is explicitly closed. It should ignore it and remain closed.